packages/parts/src/lib/settings/EditableStringField.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import type { Readable } from 'svelte/store';
+  export let t: Readable<any> = undefined;
+  import FormWrapper from '../helpers/FormWrapper.svelte';
+
+  export let attribute:string,
+    display: string = undefined,
+    required = false,
+    wrap = true,
+    minLength = 0,
+    maxlength = 30;
+  import { createEventDispatcher } from 'svelte';
+  const dispatch = createEventDispatcher<{
+    save: {attribute: string}
+  }>()
+</script>
+
+<FormWrapper action={() => dispatch('save', {attribute})} {wrap} {t} {attribute} {display}>
```

Comment:
The problem with making this component it's own form is that you are now making the form in create-dictionary.svelte useless. The enter button will no longer submit the entire form but just this one. The required attribute and related attributes like minlength and maxlength will now no longer work. On the live site when you try to submit the create-dictionary form and there is a validation problem (missing dictionary title or it's too short for example) the browser will give you the proper error message in the proper language with no work on our end. We want to keep that valuable feature as it saves us a lot of translation headache and the browser handles these errors better than our custom code could.